src/locales/translations/pt.ts

```diff
@@ -69,9 +69,9 @@ export default {
   update_modal_remind_me_next_time: "Me lembre mais tarde",
   update_modal_skip_this_update: "Ignorar esta atualização",
   gta_path_modal_cant_find_game_title:
-    "Não foi possível encontrar o GTA San Andreas!",
+    "Não foi possível encontrar o GTA: San Andreas!",
```

Comment:
I'm not sure the colon is necessary - is this used in other copy? I've only ever seen it written as "GTA San Andreas"

Reply:
This colon is not necessary, i forgot to remove on one of my updates to pt translation

Reply:
Regarding the colon, I asked why in the English language it contained the colon in the rest of the words, I also researched it and found it in the original name

https://prnt.sc/Sd4_hK9ZhcF8